### PR TITLE
Add translations

### PR DESCRIPTION
**Pull request by nogates**

### CODE REVIEW

File: frontend/app/locales/en/translations.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export default {
+  'tasks.index.title': 'View tasks',
```
Comment (atd) [inline frontend/app/locales/en/translations.js]: I am used to nested objects :smile:

**Comment by atd** [comment]
Awesome!

File: frontend/app/locales/en/translations.js (hunk 1 above)
Comment (nogates) [reply frontend/app/locales/en/translations.js]: yes! nice catch... we should be using nested objects. I thought that it didn't support it, but turns out it does. I shall change

Comment (atd) [reply frontend/app/locales/en/translations.js]: Great!

**Comment by nogates** [comment]
this is ready for review @atd

**Comment by nogates** [comment]
funny thing, seems the travis integration is not working??? I saw you have modified the `.travis.yml`, do you think it is related?

**Comment by atd** [comment]
Cool!

Regarding travis integration, I switched to restricted third-party application access policy for singularties and didn't notice I had to authorize travis.

It is working now :-) I am restarting the build of last commit so webhooks are re-sent back to this pull request

**Comment by nogates** [comment]
awesome!

**Pull request merged**